CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # HEAD
 
+## Features
+
+* **js:** Add details component and demo (#129)
+* **css:** Add footer component and demo (#184)
+
 ## Bug Fixes
 
 * **js:** Add missing del dependency in build (#271).
+* Update tokens
```

Comment:
Add a link to the tokens issue so folks can work out what changed in the update?